### PR TITLE
DEVDOCS-4680: [update] Tax Rates & Zones, more about default tax zone

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4680]

Added notes about default tax zones and zones 

[DEVDOCS-4680]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4680?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by theromulans — APPROVED** [review]
Makes sense, thanks @bc-andreadao.

Now, there are additional considerations around the default tax zone which are relevant to updating the zone (which we potentially identified after this ticket was raised, hence they are not detailed in the Jira description). Specifically:

* The client may not disable the default tax zone
* The client may not adjust the shopper target settings of the default tax zone (since it always applies to "everyone else" not otherwise covered by a different tax zone)

Unsure how we'd potentially want to describe this in the docs, and whether we'd want to do it as part of this particular changeset.

**Review by theromulans — APPROVED** [review]
Thanks @bc-andreadao!

**Review by bc-traciporter — APPROVED** [review]
LGTM!

File: docs/api-docs/tax/tax-rates-zones.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can further narrow the scope of a zone by specifying one or more customer gr
 
 <!-- theme: info -->
 > #### Default tax zone
-> A default tax zone covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone fall under the default tax zone. A default tax zone does not have `shopper_target_settings` in its requests or responses. You can't specify the customer groups or locations for a default tax zone.  
+> The default tax zone, whose zone `id` is 1, covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone fall under the default tax zone. You cannot create, disable, or delete the default tax zone. When updating the default tax zone, you cannot adjust the locations or customer groups that it covers.
```
Comment (slsriehl) [inline docs/api-docs/tax/tax-rates-zones.md:19]: 1. `whose` applies to people, not impersonal objects or idea. revise.
2. It's not just shoppers in customer groups who can be left zoneless; ungrouped customers could also have no applicable zone defined. revise.
3. You don't need to create the default tax zone, but i wouldn't put that in the same category as disabling, deleting, or mis-modifying the zone.

File: docs/api-docs/tax/tax-rates-zones.md [hunk 2]
```diff
@@ -198,6 +198,11 @@ title: Response
 
 <!-- type: tab-end -->
 
+<!-- theme: info -->
+> #### Note
```
Comment (slsriehl) [inline docs/api-docs/tax/tax-rates-zones.md:202]: make the callout title more specific. when they are called "Note" we have a bunch of duplicate anchor links on the page.

File: docs/api-docs/tax/tax-rates-zones.md (hunk 1 above)
Comment (bc-andreadao) [reply docs/api-docs/tax/tax-rates-zones.md]: @slsriehl Copyedits are made

File: docs/api-docs/tax/tax-rates-zones.md (hunk 2 above)
Comment (bc-andreadao) [reply docs/api-docs/tax/tax-rates-zones.md]: @slsriehl Callout is now specific to updating a zone

File: docs/api-docs/tax/tax-rates-zones.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ You can further narrow the scope of a zone by specifying one or more customer gr
 
 <!-- theme: info -->
 > #### Default tax zone
-> A default tax zone covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone fall under the default tax zone. A default tax zone does not have `shopper_target_settings` in its requests or responses. You can't specify the customer groups or locations for a default tax zone.  
+> The default tax zone, which has a zone `id` of 1, covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone or shoppers that don't belong in a customer group fall under the default tax zone. You cannot disable or delete the default tax zone. When updating the default tax zone, you cannot adjust the locations or customer groups that it covers.
```
Comment (slsriehl) [inline docs/api-docs/tax/tax-rates-zones.md:19]:
```suggestion
> The default tax zone, which has a zone `id` of 1, covers locations that other zones don't cover. Shoppers who don't meet the criteria for any manually-defined tax zone fall under the default tax zone. You cannot disable or delete the default tax zone. When updating the default tax zone, you cannot adjust the locations or customer groups that it covers.  
```
is there a reason that we needed to mention the customer groups at all here? I understand there's a bit of complexity re: tax zones generally and tax zones for groups, but I think that's covered elsewhere in the article.

Comment (bc-andreadao) [reply docs/api-docs/tax/tax-rates-zones.md:19]: Good point, the catch-all statement is less complicated for users, will use it

**Review by slsriehl — APPROVED** [review]
🎉

**Pull request merged**